source/extensions/transport_sockets/tls/context_config_impl.h

```diff
@@ -27,14 +29,16 @@ class ContextConfigImpl : public virtual Ssl::ContextConfig {
   const std::string& cipherSuites() const override { return cipher_suites_; }
   const std::string& ecdhCurves() const override { return ecdh_curves_; }
   // TODO(htuch): This needs to be made const again and/or zero copy and/or callers fixed.
-  std::vector<std::reference_wrapper<const TlsCertificateConfig>> tlsCertificates() const override {
-    std::vector<std::reference_wrapper<const TlsCertificateConfig>> configs;
+  std::vector<std::reference_wrapper<const Envoy::Ssl::TlsCertificateConfig>>
```

Comment:
I still see a bunch of `Envoy::Ssl` in here. Is that needed for a namespace conflict? Doesn't seem like there should be?

Reply:
Interfaces in `include/envoy` still use `Envoy:Ssl`, @bdecoste is going to rename those in a follow-up PR... Unless you prefer to do it in this one?

Reply:
My point was I don't think the `Envoy::` is required, but if this is going to be changed in a follow up we can go ahead and just merge it.

Reply:
Ah, that. I focused on `Ssl` vs `Tls` and completely missed it. Yes, it probably wasn't needed.

Reply:
Thanks for the reviews and merge. I should have the next PR to complete the directory and namespace change from ssl to tls in today or tomorrow.

Reply:
To confirm...  do you want me to leave include/envoy/ssl alone (i.e. Envoy::Ssl) or make the change to include/envoy/tls and Envoy::Tls and use fully qualified namespaces to avoid conflict?If we are sticking with ssl then I will put in a PR to remove the extraneous Envoy::

Thanks